include/global.hpp

```diff
@@ -330,32 +331,33 @@ enum Races {
     BARBARIAN = 18,
     KATARAN = 19,
     TIEFLING = 20,
+    KENKU = 21,
 
-    MAX_PLAYABLE_RACE = 21,
+    MAX_PLAYABLE_RACE = 22,
 
     // non-playable
-    LIZARDMAN = 21,
-    CENTAUR = 22,
+    LIZARDMAN = 23,
```

Comment:
Isn't this going to fuck up existing races, unless you add backwards compat?  Also about the web editor?